### PR TITLE
Proxy

### PR DESCRIPTION
**Pull request by WhiteBoardDev**

### CODE REVIEW

**Comment by WhiteBoardDev** [comment]
@cnishina can we get the readme updated with example usage commands and what situation they are used in?

File: common/settings.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            self.conf_dir = os.path.join(self.base_dir, 'conf')
+            self.log_conf = os.path.join(self.conf_dir, 'logging.conf')
+        else:
+            os_setting = None
+            if 'linux' in sys.platform:
+                os_setting = linux
+            elif 'darwin' in sys.platform:
+                os_setting = linux
+            elif 'win32' in sys.platform:
+                print 'windows'
```
Comment (WhiteBoardDev) [inline common/settings.py]: throw an exception to tell the user that its not supported yet? what os does OSX resolve to?

Comment (cnishina) [inline common/settings.py]: it resolves to linux. _troll face_ i'll fix these to throw an exception

**Comment by cnishina** [comment]
@WhiteBoardDev please review the readme.

**Pull request merged**